build.zig

```diff
@@ -160,4 +160,49 @@ pub fn build(b: *Builder) !void {
         "target remote localhost:1234",
     });
     debug_step.dependOn(&debug_cmd.step);
+
+    const zip_folder = "rpi-sdcard";
```

Comment:
Instead of zipping, could a .img file not be created that has the kernel, config file et.c inside it? Then that could be copied directly to an SD card instead of having to unzip.

Reply:
It can be done but would still require something like https://www.balena.io/etcher/ to do the copying.

Could we open an issue to improve this later?

Reply:
Oh I don't mean that the SD card should be flashed with this step, just that an image file is created, just like we do with `grub-mkrescue` for x86.

Reply:
https://forums.balena.io/t/how-do-i-create-an-sd-disk-image-file/171673

Reply:
I believe that something like this would work https://unix.stackexchange.com/questions/90793/create-iso-image-from-folder-via-terminal-commands/90794

Reply:
I took a look but it did not look like a complete solution.